src/backend/common/Pipeline.cpp

```diff
@@ -33,6 +35,12 @@ namespace backend {
             return;
         }
 
+        if (!IsCompute() && !renderPass) {
+            builder->HandleError("Pipeline render pass not set");
+            return;
+        }
+        // TODO(kainino@chromium.org): Need to verify the pipeline against its render subpass.
```

Comment:
This is a bit of work because the info is (afaik) only available in ExtractSpirvInfo, so I'm going to make it a separate patch.